src/server.jsx

```diff
@@ -70,10 +73,19 @@ const baseTag = process.env.CANON_BASE_URL === undefined ? ""
   : `
     <base href='${ BASE_URL }'>`;
 
+function getGeneratedTags() {
+  const filePath = path.join(process.cwd(), 'static/assets/index.html');
```

Comment:
this logic only works in prod. If I run `npm run dev` without having built the production build, I get an error because `index.html` has never been generated.

Reply:
I'll address this on the next commit 👍